x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/conversations/field_maps_configuration.ts

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FieldMap } from '@kbn/data-stream-adapter';
+
+export const conversationsFieldMap: FieldMap = {
+  '@timestamp': {
+    type: 'date',
+    array: false,
+    required: false,
+  },
+  users: {
+    type: 'nested',
+    array: true,
+    required: false,
+  },
+  'users.id': {
+    type: 'keyword',
+    array: false,
+    required: true,
+  },
+  'users.name': {
+    type: 'keyword',
+    array: false,
+    required: false,
+  },
+  id: {
+    type: 'keyword',
+    array: false,
+    required: true,
+  },
+  title: {
+    type: 'keyword',
+    array: false,
+    required: true,
+  },
+  is_default: {
+    type: 'boolean',
+    array: false,
+    required: false,
+  },
+  updated_at: {
+    type: 'date',
+    array: false,
+    required: false,
+  },
+  created_at: {
+    type: 'date',
+    array: false,
+    required: false,
+  },
+  messages: {
+    type: 'nested',
+    array: true,
+    required: false,
+  },
+  'messages.@timestamp': {
+    type: 'date',
+    array: false,
+    required: true,
```

Comment:
nit: We'll want to update the messages timestamp code to persist ISO8601 -- looks like it's storing local time at the moment:

<p align="center">
  <img width="500" src="https://github.com/elastic/kibana/assets/2946766/044be2fc-dd3c-4f5b-b807-0b61f6ef10f4" />
</p>

Reply:
resolved here https://github.com/elastic/kibana/pull/173487/commits/2a923e1da1c6ef513ea64e19051791ca5bb8deda